### ISSUE TITLE
Distributing JBot application

### ISSUE THREAD

**Issue by CubicDolphin** [rampatra/jbot#117]
Does JBot in any way support distribution to multiple workspaces, without having to run multiple instances?

**Comment by rampatra** [comment]
For deploying to multiple workspaces you have to first make a slack app and then JBot can be used across workspaces.

**Comment by CubicDolphin** [comment]
Yes, I have a functional application as well as two workspaces with it both installed, but where I am stuck is getting the Bot to recognize messages from both workspaces without having to run multiple instances of my application. I can retrieve from Oauth the required tokens which I can use to make Slash Commands functional and responsive, but the bot refuses to listen to any workspace not hardcoded in. Is it possible to have a single bot monitor many workspaces, or to create multiple bots dynamically?
